browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -985,6 +985,15 @@ export class CommentSection extends CanvasSectionObject {
 		this.update();
 	}
 
+	public showInNavigator(comment: Comment) {
+		const id = comment.sectionProperties.data.id - 1;
+		if (app.map._docLayer._docType === 'text') {
+			if (!document.getElementById('navigation-sidebar').classList.contains('visible'))
```

Comment:
This code is duplicated, please share by sending dispatch command:
`app.dispatcher.dispatch('showcommentsnavigator')`

Reply:
it will require argument to pass as I see, please extend:
`public dispatch(action: string) {` with optional parameter of type `data?: any`